### ISSUE TITLE
NTR, pulmo: lack of surfactant-protein C (SP-C)

### ISSUE THREAD

**Issue by GrieseAG** [obophenotype/human-phenotype-ontology#5566]
**Preferred term label:**
lack of surfactant-protein C (SP-C)

**Synonyms**
Elevated level of pro-protein C (proSP-C)

**Definition (free text, please give PubMed ID)**
Significantly decreased level or failed detection of surfactant protein C in broncho-alveolar lavage fluid. Comment: Pulmonary surfactant is a highly surface-active mixture of proteins and lipids that is synthesized and secreted onto the alveoli by type II epithelial cells. The protein part of surfactant constitutes of four types of surfactant proteins (SP), SP-A, SP-B, SP-C and SP-D. SP-A and SP-D are hydrophilic proteins that regulate surfactant metabolism and have immunologic functions, whereas SP-B and SP-C are hydrophobic molecules, which play a direct role in the organization of the surfactant structure in the interphase and in the stabilization of the lipid layers during the respiratory cycle. Lack of SP-C may result of surfactant metabolism dysfunction and is also observed in patients with other diffuse parenchymal lung diseaes, pathogenetically related to the alveolar surfactant region. 

PMID: 26375475

**Parent term (use hpo.jax.org/app)**
Abnormal biochemical composition of broncho-alveolar fluid

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM: 610913 , OMIM: 267450, OMIM: 610921 

**Gene Associations**
NKX2.1 (OMIM:  600635 ), SFTPC (OMIM: 178620), ABCA3 (OMIM: 601615)

**Your nano-attribution (ORCID)**
crapp, @mgriese, @katrinknoflach

**Comment by pnrobinson** [comment]
Absent surfactant-protein C in bronchoalveolar lavage
sibling term -- Elevated pro-surfactant-protein C level in bronchoalveolar lavage (syn Elevated proSPC in BAL)

note--PMID above is also for disease associations

**Comment by pnrobinson** [comment]
Todo defintion for Elevated pro-surfactant-protein C level in bronchoalveolar lavage
